### PR TITLE
Add release notes to the documentation

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -0,0 +1 @@
+--8<-- "CHANGELOG.md"
```
Comment (lovelydinosaur) [inline docs/release-notes.md:1]: 🤔

Comment (Kludex) [reply docs/release-notes.md:1]: With this we have the CHANGELOG on the root level, and we reference it from the documentation, and we don't duplicate the content.

**Comment by Kludex** [comment]
It's this kind of PR that I don't think it makes sense for me to wait, because I want to do more... I want to be granular on the PRs merged, and if I keep working on improvements, I would need to rebase on top of this branch, and get a list of PRs, which will take longer to be approved, and it will be more work to rebase...

This is a comment based on what we talked about "merging some PRs without review" @tomchristie

**Review by frankie567 — APPROVED** [review]
Looks sensible to me, the include pattern is clever to show the changelog in Mkdocs.

**Pull request merged**